<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WireShell</class>
 <widget class="QMainWindow" name="WireShell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>弹上配电器综合检测平台</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>测试</string>
    </property>
    <addaction name="actionAutoTest"/>
    <addaction name="actionSequenceTest"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>用户管理</string>
    </property>
    <addaction name="actionUserMange"/>
   </widget>
   <widget class="QMenu" name="menuLog">
    <property name="title">
     <string>日志管理</string>
    </property>
    <addaction name="actionLogManage"/>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menuUser"/>
   <addaction name="menuLog"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAutoTest">
   <property name="text">
    <string>自动测试</string>
   </property>
  </action>
  <action name="actionSequenceTest">
   <property name="text">
    <string>序列测试</string>
   </property>
  </action>
  <action name="actionUserMange">
   <property name="text">
    <string>用户管理</string>
   </property>
  </action>
  <action name="actionLogManage">
   <property name="text">
    <string>日志管理</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
